cmd/build/command.go

```diff
@@ -129,3 +149,43 @@ func validateDockerfile(file string) error {
 	_, _, err = instructions.Parse(parsedDockerfile.AST)
 	return err
 }
+
+func (*Command) loadContext(ctx context.Context, options *types.BuildOptions) error {
+	ctxOpts := &contextCMD.ContextOptions{}
+
+	// before calling the context command, there is need to retrieve the context and
+	// namespace through the given manifest. If the manifest is a Dockerfile, this
+	// information cannot be extracted so call to GetContextResource is skkiped.
+	if err := validateDockerfile(options.File); err != nil {
+		ctxResource, err := model.GetContextResource(options.File)
+		if err != nil {
+			return err
+		}
+
+		if options.Namespace != "" {
```

Comment:
Nit: We need to do some validation between `options.Namespace` and the ones retrieved from the manifest. If we don't use [LoadContextFromPath](https://github.com/okteto/okteto/blob/3636552ee541450e82c1904ba58a097b839e75c6/cmd/context/utils.go#L231), can we update `ctxResource` with
```golang
if err := ctxResource.UpdateNamespace(namespace); err != nil {
    return err
}

if err := ctxResource.UpdateContext(k8sContext); err != nil {
    return err
}
```
and after that set the options with the content of `ctxResource`.

Those validations prevents the user to have a different context specified on the flag and the manifest